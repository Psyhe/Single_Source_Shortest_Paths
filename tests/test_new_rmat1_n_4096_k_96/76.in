4096 3256 3297
3256 256 40
3256 552 9
3256 2 76
3264 516 19
3264 1220 71
3264 672 199
3264 195 112
3264 65 52
3264 1026 134
3264 3200 5
3264 100 63
3264 20 114
3264 32 165
3264 209 84
3264 2048 48
3264 460 25
3264 82 94
3264 0 120
3264 843 189
3264 8 33
3264 0 158
3264 10 22
3264 3488 107
3264 516 230
3264 556 235
3265 2 233
3265 2052 196
3265 37 102
3265 265 35
3265 13 222
3265 1536 233
3265 1798 156
3265 1344 210
3266 8 116
3266 128 174
3266 144 94
3266 2324 83
3266 0 112
3266 400 253
3266 13 7
3266 76 43
3266 56 143
3266 8 184
3266 1 225
3266 514 115
3267 2432 248
3267 656 64
3267 6 225
3268 2576 169
3268 516 43
3268 296 119
3268 264 88
3268 532 156
3269 0 127
3269 1568 3
3269 2992 78
3270 16 95
3270 16 174
3272 1088 249
3272 1608 104
3272 60 184
3272 64 146
3272 2112 179
3272 3222 23
3273 3648 208
3273 20 105
3273 2624 159
3274 362 46
3274 2564 179
3274 0 58
3276 2304 237
3276 0 146
3277 392 58
3278 1031 109
3278 0 79
3278 50 204
3278 1 147
3280 16 78
3280 1 100
3280 2564 105
3280 0 129
3280 2065 87
3280 1057 100
3280 835 40
3280 898 208
3280 162 25
3282 297 113
3282 2604 36
3282 3361 128
3283 1024 51
3283 2564 208
3284 1090 126
3284 168 131
3284 512 23
3285 802 76
3285 270 61
3285 392 209
3286 1032 26
3288 4 193
3288 2048 217
3288 1666 74
3288 128 29
3288 48 44
3288 266 106
3289 13 97
3291 2827 190
3293 1024 141
3296 4 133
3296 2209 128
3296 788 216
3296 2384 233
3296 1027 102
3296 1689 49
3296 265 20
3296 804 150
3296 262 32
3297 1569 81
