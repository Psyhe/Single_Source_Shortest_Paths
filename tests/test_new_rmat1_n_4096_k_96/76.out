10
inf
inf
inf
inf
inf
inf
inf
7
47
21
69
45
27
97
inf
91
107
58
inf
146
62
79
inf
46
inf
86
52
27
87
28
inf
32
111
inf
195
inf
142
inf
inf
32
87
